core/src/data/value/into.rs

```diff
@@ -13,7 +13,7 @@ use {
 };
 
 impl From<&Value> for String {
-    fn from(v: &Value) -> Self {
+    fn from(v: &Value) -> String {
```

Comment:
It's not open, but I'll leave a comment. :)

<https://doc.rust-lang.org/stable/std/convert/trait.From.html>

`From` is the implementation of the above std Trait. `Self` and `String` are the same here, but it seems good to leave it semantically converted to `Self`.

example) <https://doc.rust-lang.org/rust-by-example/trait.html>

Examples from other famous libraries)
- <https://github.com/serde-rs/serde/blob/ce0844b9ecc32377b5e4545d759d385a8c46bc6a/test_suite/tests/test_remote.rs#L160-L182>